### PR TITLE
Update git-init images

### PR DESCRIPTION
**Pull request by mhjacks**
However, the new git-init image also seems to have a bug with initially created working trees, such that explicit checkout of the branch is called for.

Extra commits pushed since the original:
1) Experimenting with different argocd server name.  Reverted to the one we were using.
2) Added an explicit resource to create the necessary rolebinding to allow manuela-ci to push images to manuela-tst-all

### CODE REVIEW

**Comment by claudiol** [comment]
Ok.  I reviewed changed and they look ok.  Told @mhjacks about changing his repos in quay.io to public.  Will go ahead and merge these in.

**Comment by beekhof** [comment]
@mhjacks lets try and drop the image stream changes before creating future PRs

**Comment by mhjacks** [comment]
ACK. #23 should be an improvement on this front.

File: common/tekton/tasks/github-push.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
     default: ""
   steps:
   - name: push
-    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.11.0
+    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.24.3
```
Comment (beekhof) [inline common/tekton/tasks/github-push.yaml:27]: I feel like we'd ideally have this specified in a central location

**Pull request merged**